packages/icon/src/vars/index.js

```diff
@@ -1,8 +1,8 @@
 import core from '@pluralsight/ps-design-system-core'
 
-import icons from '../js/icons'
+import fileNames from '../react/file-names'
```

Comment:
I feel like this is a good reason for file-names to remain in `/js`...  `vars` are not going to be framework agnostic, as they're meant to be (so can be reused across side-by-side js framework tech and the css, if they are pulling in react dependencies.

Perhaps what file-names includes now needs to split -- the general use bits for the file-name collection could go back into `js/` and the react component template could stay in the new `react/` location.

Reply:
i see you point. i'll move it back to `js` for now. we can readdress a better approach in the future